rules/max-one-break-per-loop.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @fileoverview Flags any loops which contain two or more break statements
+ * @author Artem Litchmanov <artem.litchmanov@gmail.com>
+ */
+
+'use strict';
+
+module.exports = {
+
+    meta: {
+
+        docs: {
+            recommended: true,
+            type: 'error',
+            description: 'Flag all loops which contain two or more break statements'
+        },
+
+        schema: []
+
+    },
+
+    create: function (context) {
+
+        var allBreakStatementDeclarations = 0;
+
+        function inspectBreakStatement(emitted) {
+
+            if (!emitted.exit) {
+                allBreakStatementDeclarations += 1;
+            }
+        }
+
+        //While exiting for loop Node, Report if Break statement declarations is more than 1
+        function inspectLoopStatement(emitted) {
```

Comment:
I **think** this technique would break if we have multiple loops with break statements.
Could you please add below test and see if its passing

Acceptance
```
`contract Foo { function bar() { do { break; } while (1);  for(uint i = 0; i < 10; i++) { break; } } }`
```
Notice multiple loops having break st.
Why I say this is because if we have multiple loops, the **`!emitted.exit` block of all the loops gets executed first** before their respective `emitted.exit` blocks get executed (which, I think, is not what you intended).

If the above fails, let me know and I'll describe a strategy that ensures 0 false positives